src/style.css

```diff
@@ -0,0 +1,23 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
don't add these properties to `*` selector - reset it for `body` (and/or smth else) if you need